### PR TITLE
Be/project instructions

### PR DESCRIPTION
**Pull request by beccaelenzil**
Moved instructions for each Wave from Activity directions in Learn to the README.

Learn changes in the PR: https://github.com/Ada-Developers-Academy/core-unit-2/pull/82

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,29 +19,40 @@ Here is a minimal setup for a Flask API.
...
+    * Make regular commits
+    * Push commits before switching driver
+    * Pull before starting to drive
+
 ## Guidelines for Pair-Programming
 
 - The driver is the person who is at the keyboard and mouse
 - The navigator is the person who is thinking out loud, actively collaborating with the driver about the next step, and helping guide the development
-- Trade-off driver and navigator roles often
+- Trade-off driver and navigator roles often, at least daily
```
Comment (CheezItMan) [inline README.md:36]: Daily?  Maybe instead each hour.

**Review by CheezItMan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (beccaelenzil) [reply README.md:36]: Students generally work on this for 1-2 hours. What about this:
```suggestion
- Trade-off driver and navigator roles often, at least daily, or every hour for longer work sessions.
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+## Goal
 
-## Github Setup
+Our coding skills improve with repetition.
+
+Our goal is to practice creating a minimal Flask API in a pair or group setting.
+
+Pairing with other programmers will help create stronger understanding of the material, and improved skill at working with others. Software teams thrive on collaboration, so working side-by-side with someone while coding is vital!
+
+In this activity, we will build a Solar System API. This API will store information about different planets.
```
Comment (CheezItMan) [inline README.md:11]: I just prefer a bulleted list of goals (feel free to ignore):

```suggestion

* Improve our understand of Flask & SQL Alchemy with repetition
* Discuss and explain Flask code together in pair or group programming
* Improve skill at working with others. 
    * Software teams thrive on collaboration, so working side-by-side with someone while coding is vital!

In this activity, we will build a Solar System API. This API will store information about different planets.
```

File: README.md (hunk 1 above)
Comment (CheezItMan) [inline README.md]: I think people get better outcomes switching more often than daily, but don't have data on that.

File: project-directions/wave_04.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Wave 04: Handling 404s
```
Comment (CheezItMan) [inline project-directions/wave_04.md:1]: So they'll have a 404 day.  🤣 , a day not found

File: project-directions/wave_05.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Complete the following requirements, with similar functionality to the Hello Books API:
+
+1. Create a `.env` file.
+1. Populate it with two environment variables: `SQLALCHEMY_DATABASE_URI` and `SQLALCHEMY_TEST_DATABASE_URI`. Set their values to the appropriate connection strings.
+1. Create a test database with the correct, matching name.
+1. Refactor the `create_app` method to:
+   * Check for a configuration flag
+   * Read the correct database location from the appropriate environment variables
+1. Manually test that our development environment still works.
+1. Create a `tests` folder with the files `tests/__init__.py`, `tests/conftest.py`, and `tests/test_routes.py`.
```
Comment (CheezItMan) [inline project-directions/wave_05.md:15]: Just makes it easier to read/check-off

```suggestion
1. Create a `tests` folder with the files:
    -  `tests/__init__.py`
    -  `tests/conftest.py`
    -  `tests/test_routes.py`.
```

**Review by CheezItMan — APPROVED** [review]
Overall looks good.

**Pull request merged**